Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -21,50 +21,24 @@
 ; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 ; THE POSSIBILITY OF SUCH DAMAGE.
 
-(version 1)
-(deny default (with partial-symbolication))
-(deny nvram*)
-(deny system-privilege)
-(allow system-audit file-read-metadata)
+#include "Shared/Sandbox/iOS/common.sb"
 
-;; Silence spurious logging due to rdar://20117923 and rdar://72366475
-(deny system-privilege (privilege-id PRIV_GLOBAL_PROC_INFO) (with no-report))
+#if USE(SANDBOX_VERSION_3)
+(allow system-fcntl
+    (fcntl-command
+        F_ADDFILESIGS_RETURN
+        F_CHECK_LV
+        F_GETPATH))
+
+(with-filter (mac-policy-name "Sandbox")
+    (allow system-mac-syscall (mac-syscall-number 5)))
+#endif
 
 ;;;
 ;;; The following rules were originally contained in 'common.sb'. We are duplicating them here so we can
 ;;; remove unneeded sandbox extensions.
 ;;;
```

Comment:
Perhaps this comment should move to `Shared/Sandbox/iOS/common.sb`, too?

Reply:
There are still quite a few rules that were not moved to Shared/Sandbox/iOS/common.sb in this patch, so I think we may leave the comment for now. However, we should move all the common rules to the new file including the comment. I will address that in a separate patch.

Thanks for reviewing!